app/models/student_application.rb

```diff
@@ -11,8 +11,6 @@
 #
 
 class StudentApplication < ActiveRecord::Base
-  include AssignToCurrentSemester
```

Comment:
Nope! I didn't really like it since the way semester_id gets set isn't clear (it's pretty much magically set from the controller's perspective). Also, it's been a source of errors for everyone so far in terms of running tests / playing well with FactoryGirl. I decided to just explicitly set the current semester everywhere.

See https://github.com/calblueprint/calblueprint.org/blob/qd/npo-apps/app/controllers/nonprofit_applications_controller.rb#L20-L24 for how that gets set.